### DISCUSSION TITLE
One to Many Relationship on External Source

### DISCUSSION THREAD

**Discussion by sleepsterz** [Budibase/budibase#2197] [Help] [answered]
When trying to make a relationship with the new relationships menu the only options that are given are one and many. 
I have tried one to see if it could be configured to be a one to many relationship but was unable to get it to work and as far as I can tell using the many relationship needs a bridge table. Is there something I am missing?

**Comment by shogunpurple** [comment, marked as answer]
Hi @sleepsterz - we are currently looking into this one https://github.com/Budibase/budibase/issues/2167

You can track it for progress. Going to close this one for now as a dupe

**Reply by sleepsterz** [reply]
Alright thanks for the prompt response.
